packages/server/modules/activitystream/services/streamActivityService.ts

```diff
@@ -137,10 +143,29 @@ async function addStreamInviteDeclinedActivity({
   })
 }
 
-module.exports = {
-  addStreamPermissionsAddedActivity,
-  addStreamPermissionsRevokedActivity,
-  addStreamInviteAcceptedActivity,
-  addStreamInviteSentOutActivity,
-  addStreamInviteDeclinedActivity
+/**
+ * Save "user mentioned in stream comment" activity item
+ */
+export async function addStreamCommentMentionActivity(params: {
+  streamId: string
+  mentionAuthorId: string
+  mentionTargetId: string
+  commentId: string
+  threadId: string
+}) {
+  const { streamId, mentionAuthorId, mentionTargetId, commentId, threadId } = params
+  await saveActivity({
+    streamId,
+    resourceType: ResourceTypes.Comment,
+    resourceId: commentId,
+    actionType: ActionTypes.Comment.Mention,
+    userId: mentionTargetId,
```

Comment:
This should be the mentionAuthorId, we're recording the activities of the user who made the activity.

Reply:
good catch, i'll fix it

Reply:
fixed